fluidsynth.py

```diff
@@ -693,6 +703,8 @@ def __init__(self, gain=0.2, samplerate=44100, channels=256, **kwargs):
         self.midi_driver = None
         self.router = None
         self.custom_router_callback = None
+    def __del__(self):
+        self.delete()
```

Comment:
Can we remove this change and see if the tests pass?
```suggestion
```

Reply:
I'm sorry. #86 and my latest update in this pr shows that you're right. It may caused by deleting itself multi times.

Reply:
Once this is merged, it might be a good idea to re-add `.__del__()` in a separate pull request and try to fix the failing tests to gracefully deal with its presence.